src/cpg_flow/targets/sequencinggroup.py

```diff
@@ -0,0 +1,187 @@
+from typing import Optional
+
+from cpg_utils import Path
+from cpg_utils.config import reference_path
+
+from cpg_flow.filetypes import AlignmentInput, BamPath, CramPath, FastqPairs, GvcfPath
+from cpg_flow.metamist import Assay
+from cpg_flow.targets import PedigreeInfo, Sex, Target
+
+
+class SequencingGroup(Target):
+    """
+    Represents a sequencing group.
+    """
+
+    def __init__(
+        self,
+        id: str,  # pylint: disable=redefined-builtin
```

Comment:
Maybe outside of scope for this, but what do we think of renaming this param haha, don't love when we ignoring pylint and when we re-define protected names.

Reply:
I had a look at pylint doesn't throw any errors when I remove that ignore comment. Guess there's no need to rename it then hehe